docs/user_guide/model_configuration.md

```diff
@@ -1027,18 +1027,15 @@ for examples on specifying different variants of warmup samples.
 ## Response Cache
 
 The model configuration `response_cache` section has an `enable` boolean used to
-enable the Response Cache for this model. In addition to enabling the cache in
-the model config, a non-zero `--response-cache-byte-size` must be set when
-starting the server.
+enable the Response Cache for this model. 
 
 ```
 response_cache {
-  enable: True
+  enable: true
 }
 ```
 
-See the [Response
-Cache](response_cache.md)
-and [ModelConfig
-protobuf](https://github.com/triton-inference-server/common/blob/main/protobuf/model_config.proto).
-docs for more information.
+In addition to enabling the cache in the model config, a `--cache-config` must
```

Comment:
Is it possible to have a default value for local cache to improve the UX and remove one additional step?

Reply:
The old behavior was to opt-in on server-side via a non-zero `--response-cache-byte-size` arg, so I tried to mimic that with the `--cache-config` arg rather than require some extra `--enable/disable-cache` flag for server-wide control.

I'd rather keep the same behavior for now, and consider a default server-side choice if there was user complaint. If we wanted to remove this step, we'd need a bit more logic to determine if there is at least one model being served with `response_cache { enable: true }` in its config to allocating a chunk of memory for the cache unnecessarily.

Reply:
Makes sense. Resolving.